src/_guides/packages.md

```diff
@@ -0,0 +1,227 @@
+---
+title: How to use packages
+short-title: Packages
+description: Learn more about pub, Dart's tool for managing packages.
+---
+
+The Dart ecosystem uses _packages_ to manage shared software
+such as libraries and tools.
+To get Dart packages, you use the **pub package manager**.
+You can find publicly available packages on the [**Pub site**,]({{site.pub}})
+or you can load packages from the local file system or elsewhere,
+such as git repositories.
+Wherever your packages come from, pub manages version dependencies,
+helping you get package versions that work with each other and
+with your SDK version.
+
+Most [Dart-savvy IDEs][] offer support for using pub that
+includes creating, installing, updating, and publishing packages.
+Or you can use [`pub` on the command line](/tools/pub/cmd).
+
+{% include flutter-packages.md %}
+
+At a minimum,
+a Dart package is a directory containing a [pubspec file](/tools/pub/pubspec).
+The pubspec contains some metadata about the package. Additionally,
+a package can contain dependencies (listed in the pubspec),
+Dart libraries, apps, resources, tests, images, and examples.
+
+<aside class="alert alert-info" markdown="1">
+  **Note:** If your Dart app depends on one or more packages,
+  then your app itself must be in a package.
+</aside>
+
+To use a package, do the following:
+
+* Create a pubspec (a file that lists package dependencies and includes
+  other metadata, such as a name for your package).
+* Use pub to get your package's dependencies.
+* If your Dart code depends on a library in the package, import the library.
+
+## Creating a pubspec
+
+The pubspec is a file named <code class="literal">pubspec.yaml</code>
+that's in the top directory of your application.
+The simplest possible pubspec lists only the package name:
```

Comment:
Maybe expand to '...only the package name of the app'?

Reply:
The context is broader in this sentence, and I'd have to make it a lot more specific if I add those words.